app/javascript/dashboard/assets/scss/plugins/_multiselect.scss

```diff
@@ -48,7 +48,7 @@
         overflow: hidden;
         text-overflow: ellipsis;
         white-space: nowrap;
-        width: 100%;
+        width: fit-content;
```

Comment:
These changes will fix the width issue on the label report page.
<img width="503" alt="Screenshot 2023-02-23 at 8 22 23 PM" src="https://user-images.githubusercontent.com/64252451/220943304-adc0226f-6e65-4621-8c51-24944f0c648c.png">